src/js/util.js

```diff
@@ -17,5 +17,69 @@ define(function (require, exports) {
   exports.track = function(event, data) {
     Intercom('trackEvent', event, data);
   };
+  
+  // Take an array of URLs and return a template of the form
+  // `//{s}.foo.bar/tiles/baz` and an array of substitutions for `s` that will
+  // generate the original URLs
+  exports.templatizeURLs = function templatizeURLs(urls) {
```

Comment:
Ah! I think I misunderstood this step -- it's before using the built-in leaflet tools. I'll re-read next week and try to come up with a more general arch question. The note about testing still holds.